javascript/ql/test/library-tests/frameworks/NodeJSLib/tests.ql

```diff
@@ -2,7 +2,7 @@ import isCreateServer
 import RequestInputAccess
 import RouteHandler_getAResponseHeader
 import HeaderDefinition_defines
-import SystemCommandExecution
+import SystemCommandExecutions
```

Comment:
I changed this because the previous name generated a warning in the test output, that it clashed with the module of the same name in `ConceptsSpecific.qll`.